lib/errors.ts

```diff
@@ -37,4 +37,4 @@ export class JellyfishNoView extends BaseTypedError {}
 export class JellyfishPermissionsError extends BaseTypedError {}
 export class JellyfishSchemaMismatch extends BaseTypedError {}
 export class JellyfishSessionExpired extends BaseTypedError {}
-export class JellyfishUnknownCardType extends BaseTypedError {}
+export class JellyfishUnknownContractType extends BaseTypedError {}
```

Comment:
Another breaking change. Isolate these into another PR. This is not the right time for that.